.dev/docker/default.conf

```diff
@@ -0,0 +1,51 @@
+# If using Ubuntu this file should be placed in:
+# 			/etc/nginx/sites-available/
+#
+# If using CentOS this file should be placed in:
+#				 /etc/nginx/conf.d/
+#
+server {
+    listen 80;
+    server_name _;
+    
+    root /app/public;
+    index index.html index.htm index.php;
+    charset utf-8;
+
+    location / {
+        try_files $uri $uri/ /index.php?$query_string;
+    }
+
+    location = /favicon.ico { access_log off; log_not_found off; }
+    location = /robots.txt  { access_log off; log_not_found off; }
+
+    access_log off;
+    error_log  /var/log/nginx/pterodactyl.app-error.log error;
+
+    # allow larger file uploads and longer script runtimes
+    client_max_body_size 100m;
+    client_body_timeout 120s;
+  
+    sendfile off;
+
+    location ~ \.php$ {
+        fastcgi_split_path_info ^(.+\.php)(/.+)$;
+        # the fastcgi_pass path needs to be changed accordingly when using CentOS
+        fastcgi_pass unix:/var/run/php/php-fpm7.2.sock;
+        fastcgi_index index.php;
+        include fastcgi_params;
+        fastcgi_param PHP_VALUE "upload_max_filesize = 100M \n post_max_size=100M";
+        fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+        fastcgi_param HTTP_PROXY "";
+        fastcgi_intercept_errors off;
+        fastcgi_buffer_size 16k;
+        fastcgi_buffers 4 16k;
+        fastcgi_connect_timeout 300;
+        fastcgi_send_timeout 300;
+        fastcgi_read_timeout 300;
+    }
+
+    location ~ /\.ht {
+        deny all;
+    }
```

Comment:
I would change this rules to block access to all files and folder that begin with dot:
```
    location ~ /\. {
        access_log off;
        log_not_found off; 
        deny all;
}
```

Screenshot of latest release where I have highlighted the . files and folder:
![image](https://user-images.githubusercontent.com/858296/42461084-ecbceacc-839f-11e8-9f7d-cc18ef397641.png)